packages/core/src/plugin/index.ts

```diff
@@ -20,7 +19,6 @@ export async function resolveFarmPlugins(config: UserConfig) {
   }
 
   const rustPlugins = [];
-
   const jsPlugins: JsPlugin[] = [];
 
   for (const plugin of plugins) {
```

Comment:
This change should support async plugin, that is, the plugin may be a promise, so we should judge here whether it is a promise received, wait for it to resolve, and then judge whether it is a rust plugin or a js plugin.

> But the above implementation alone does not support nesting, so it would be better if you could help implement nesting.